crates/wysiwyg/src/dom/parser/parse.rs

```diff
@@ -125,7 +125,14 @@ mod sys {
                         // Special case for code block, translate '\n' into <br /> nodes
                         let is_inside_code_block =
                             self.current_path.contains(&CodeBlock);
-                        convert_text(&text.content, node, is_inside_code_block);
+                        let is_only_child_in_parent =
+                            node.children().is_empty();
```

Comment:
I'm confused at how `is_only_child_in_parent` is being calculated from `node.children().is_empty()`. Should we not look at the size of `panode.children`?